universum/modules/automation_server/github_server.py

```diff
@@ -0,0 +1,58 @@
+from ..error_state import HasErrorState
+from ...lib.module_arguments import ModuleArgumentParser
+from .base_server import BaseServerForHostingBuild, BaseServerForTrigger
+
+__all__ = [
+    "GithubServer",
+]
+
+
+class GithubServer(BaseServerForHostingBuild, BaseServerForTrigger, HasErrorState):
+    @staticmethod
+    def define_arguments(argument_parser: ModuleArgumentParser) -> None:
+        parser = argument_parser.get_or_create_group("Github Actions variables",
+                                                     "Github Actions specific parameters")
+
+        parser.add_argument("--github-server", "-ghs", dest="server_url", metavar="GITHUB_SERVER_URL",
+                            help="Github server URL, used on Github Actions for constructing links to builds. "
+                                 "For example: https://github.com", type=str)
+        parser.add_argument("--github-repo", "-ghr", dest="repo", metavar="GITHUB_REPOSITORY",
+                            help="Github owner and repository name, used on Github Actions for constructing links to "
+                                 "builds. For example: octocat/Hello-World", type=str)
+        parser.add_argument("--github-run-id", "-ghri", dest="run_id", metavar="GITHUB_RUN_ID",
```

Comment:
For me it is kind of hard to understand this parameter without some page that describes using of universum in GitHub Actions. Is it supposed to be obtained from some GA parameters during launch? Which one? What is the recommended command line then?

Reply:
All this parameters are getting from GA environment variables, and their are not recommended  to be set on the command line.